traefik/values.yaml

```
@@ -413,6 +413,10 @@ metrics:
     ## When manualRouting is true, it disables the default internal router in
     ## order to allow creating a custom router for prometheus@internal service.
     manualRouting: false
+    ## Add HTTP header labels to metrics
+    ## Example: headerLabels:
+    ##            label: headerKey
```

```suggestion
    ## Add HTTP header labels to metrics. See EXAMPLES.md or upstream doc for usage.
```
values.yaml is already (very) long, don't you think ?